examples/tutorials/analysis-1d/cta_sensitivity.py

```diff
@@ -83,30 +95,34 @@
     "$GAMMAPY_DATA/cta-1dc/caldb/data/cta/1dc/bcf/South_z20_50h/irf_file.fits"
 )
 location = observatory_locations["cta_south"]
-pointing = SkyCoord("0 deg", "0 deg")
-livetime = 5.0 * u.h
+
+livetime = 50.0 * u.h
 obs = Observation.create(
-    pointing=pointing, irfs=irfs, livetime=livetime, location=location
+    pointing=pointing_info, irfs=irfs, livetime=livetime, location=location
 )
 
 spectrum_maker = SpectrumDatasetMaker(selection=["exposure", "edisp", "background"])
 dataset = spectrum_maker.run(empty_dataset, obs)
 
-
 ######################################################################
-# Now we correct for the energy dependent region size:
+# Now we correct for the energy dependent region size.
 #
+# **Note**: In the calculation of the containment radius, we use the point spread function
+# which is defined dependent on true energy to compute the correction we apply in reconstructed
+# energy, thus neglecting the energy dispersion in this step.
 
 containment = 0.68
 
 # correct exposure
 dataset.exposure *= containment
 
 # correct background estimation
+# Warning: this neglects the energy dispersion by computing the containment
+# radius from the PSF in true energy but using the reco energy axis.
```

Comment:
Why are you speaking about reco energy? the containment_radius of the PSF is in true energy, is not it?

Reply:
Yes, that's the issue. This uses the PSF to compute a containment radius in true energy that is then applied to reco energy bins of the background to "simulate" the effect of rad max.

The example is computing something in true energy and then applies it in reco energy.